prow/plank/plank_test.go

```diff
@@ -510,6 +516,7 @@ func TestSyncKubernetesJob(t *testing.T) {
 			expectedPodHasName: true,
 			expectedNumPods:    1,
 			expectedReport:     true,
+			expectedURL:        "NEWPOD/pending",
```

Comment:
yeah exactly..was going to mention this in the other comments but glad you caught it.

I mean we could even have it live on the client rather than a global...I'm not sure...but I'll create the issue.